### DISCUSSION TITLE
Add ability to make POST APIs

### DISCUSSION THREAD

**Discussion by chandradharrao** [apiffer-apis/apiffer#9] [Ideas]
My idea for the implementation:

1. Create an API in server to accept POST API creation
2. Ask user to enter a MongoDB Atlas(any cloud storage or store in User's browser using IndexedDB) URL to store the POST request data
3. Ask user for the schema of incoming data
4. Store the data received in the MongoDB Atlas URL(or Indexed DB) provided

**Comment by ShreyamMaity** [comment]
hmm we're planning to add a drag-and-drop UI to build the post logic

**Comment by chandradharrao** [comment]
Okay, how are you planning on storing the data in the backend sent to the POST request?
Are you not going to store the data and just ignore it, since anyway it is not queried again?

**Comment by ShreyamMaity** [comment]
our initial plan is to make this a vs code extension (which is under progress) and then you can run APIs on your local system and then we don't have to worry about scaling the product. And then we're going to use local MongoDB to store the post-req data. And for the website, we plan to use mongo image (docker).

**Comment by ShreyamMaity** [comment]
we have to change the current backend creation logic for that. I am thinking of a better way to execute the solution. And have some planning like exporting backend code whenever you're done with your testing or maybe one-click hosting to Heroku.(But now heroku is not free anymore so have to plan for other free services)

**Comment by chandradharrao** [comment]
Ahh I see.
Making it a VS code extension is a good idea.
Will close the issue then.
Thanks for your time.

**Comment by ShreyamMaity** [comment]
converted this to a discussion so that others can share their thoughts.
